src/source_inspector/symbols_tree_sitter.py

```diff
@@ -190,6 +190,11 @@ def traverse(node, symbols, strings, language_info):
         "identifiers": ["identifier"],
         "string_literals": ["raw_string_literal"],
     },
+    "Swift": {
+        "wheel": "tree_sitter_swift",
+        "identifiers": ["identifier"],
+        "string_literals": ["raw_string_literal"],
```

Comment:
After going through https://github.com/alex-pinkus/tree-sitter-swift/tree/main/queries, it seems like we will get better symbols and strings if we use these settings for filtering nodes.

```suggestion
        "identifiers": ["simple_identifier"],
        "string_literals": ["line_str_text"],
```

Reply:
Thanks, modified these accordingly.